ohara-manager/client/src/components/Workspace/Edit/Settings/Nodes/NodeCard.js

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2019 is-land
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'classnames';
+import { map, reduce } from 'lodash';
+
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Collapse from '@material-ui/core/Collapse';
+import Grid from '@material-ui/core/Grid';
+import IconButton from '@material-ui/core/IconButton';
+import Typography from '@material-ui/core/Typography';
+
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+
+import { useWorkspace } from 'context';
+import ResourceItem from './ResourceItem';
+import ServiceItem from './ServiceItem';
+import ServiceSwitch from './ServiceSwitch';
+
+import { Wrapper } from './NodeCardStyles';
+
+const NodeCard = ({ node }) => {
+  const { currentWorker, currentBroker, currentZookeeper } = useWorkspace();
+
+  const [expanded, setExpanded] = React.useState(false);
+  const handleExpandClick = () => {
+    setExpanded(!expanded);
+  };
+
+  const flatServices = services => {
```

Comment:
The verb form of the word _flat_ :)

```suggestion
  const flattenServices = services => {
```